src/layout/IHyprLayout.cpp

```diff
@@ -193,7 +193,11 @@ bool IHyprLayout::onWindowCreatedAutoGroup(PHLWINDOW pWindow) {
     if (pWindow->m_bIsFloating && !OPENINGON->m_bIsFloating)
         denied = true;
 
-    if (*PAUTOGROUP                                      // check if auto_group is enabled.
+    bool swallowing = false;
+    if (pWindow->m_pSwallowed || pWindow->m_bGroupSwallowed)
+        swallowing = true;
```

Comment:
`const bool SWALLOWING = pWindow->m_pSwallowed || pWindow->m_bGroupSwallowed;` bud..?

Reply:
Thanks. I have also changed the other variables too for consistency.